scripts/get_build_deps.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Add the package sources
+#wget -qO - https://packages.irods.org/irods-signing-key.asc | sudo apt-key add -
+#echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/renci-irods.list
+sudo apt-get update
+sudo apt install -y irods-dev irods-externals-\* irods-icommands
+
+# log in so we can grab the up to date irods lol
+cat bison_credentials | iinit
```

Comment:
Is this documented anywhere?
Is this something the user deploying this software needs to know about?

Reply:
This is only temporary.   It’s where we currently have a build of main (pre 4.3.1) because of switch user.

Reply:
I'll mention that and indicate what should be cut out.